### PR TITLE
fix: add isRealNameVerified method and update readme

### PR DESCRIPTION
**Pull request by yugasun**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -383,45 +384,29 @@ tencentLogin.getResult();
...
-  async getUserAuth() {
-    const getUserAuthInfo = new GetUserAuthInfo();
-    const uin = 123456787890;
-    console.log(await getUserAuthInfo.isAuth(uin));
-  }
-}
-
-const userAuthInfo = new UserAuthInfo();
-userAuthInfo.getUserAuth();
+const isAuth = await isRealnameAuth(secretId, secretKey);
```
Comment (timqian) [inline README.md:394]: Should it be `isRealnameAuth({ secretId, secretKey });` ?

Comment (timqian) [reply README.md:394]: Also, do you think `isRealNameAuth` or `isRealNameVerified` will be a better name for this method?

File: test/isRealnameAuth.test.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+require('dotenv').config();
+const { isRealnameAuth } = require('../sdk/account');
+
+async function main() {
+  const res = await isRealnameAuth({
+    secretId: process.env.TENCENT_SECRET_ID,
+    secretKey: process.env.TENCENT_SECRET_KEY,
+    // token: process.env.TENCENT_TOKEN,
```
Comment (timqian) [inline test/isRealnameAuth.test.js:10]: Question: When will we need to pass this token?

Comment (yugasun) [reply test/isRealnameAuth.test.js:10]: When developer use Wechat scanning to login for `sls deploy`.

File: README.md (hunk 1 above)
Comment (yugasun) [reply README.md]: Good idea.

File: README.md [hunk 3]
```diff
@@ -383,45 +384,29 @@ tencentLogin.getResult();
...
-  async getUserAuth() {
-    const getUserAuthInfo = new GetUserAuthInfo();
-    const uin = 123456787890;
-    console.log(await getUserAuthInfo.isAuth(uin));
-  }
-}
-
-const userAuthInfo = new UserAuthInfo();
-userAuthInfo.getUserAuth();
+const isVerified = await isRealNameVerified(secretId, secretKey);
```
Comment (timqian) [inline README.md:394]: `isRealNameVerified({secretId, secretKey})`

File: README.md [hunk 4]
```diff
@@ -383,45 +384,29 @@ tencentLogin.getResult();
 
 通过此接口，可以判断用户是否在腾讯云实名认证
 
-基本使用方法(GetUserAuthInfo)：
+基本使用方法(isRealnameAuth)：
```
Comment (timqian) [inline README.md:387]: `isRealNameVerified`

**Review by timqian — CHANGES_REQUESTED** [review]

**Review by timqian — APPROVED** [review]

**Pull request merged**